packages/tools/devtools/devtools-core/src/data-visualization/DefaultVisualizers.ts

```diff
@@ -245,13 +246,28 @@ export const visualizeSharedTree: VisualizeSharedObject = async (
 	visualizeChildData: VisualizeChildData,
 ): Promise<FluidObjectTreeNode> => {
 	const sharedTree = sharedObject as ISharedTree;
-	const content = sharedTree.contentSnapshot();
+	const contentSnapshot = sharedTree.contentSnapshot();
+
+	// Root node of the SharedTree's treeview. Assume there is only one root node.
+	const treeView = contentSnapshot.tree[0];
+
+	// Schema of the tree node.
+	const treeSchema = contentSnapshot.schema.nodeSchema.get(treeView.type);
+
+	// Traverses the SharedTree and generates a visual representation of the tree (mainly composed of `schema` and `fields` field).
```

Comment:
Nit (I don't think this is necessary, and might be incorrect now?)
```suggestion
	// Traverses the SharedTree and generates a visual representation of the tree and its schema.
```